documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -163,6 +163,11 @@ Specifies a subdirectory/prefix from which to load overlays - defaults to `overl
 
 NOTE: Unless `+${os_prefix}${overlay_prefix}README+` exists, overlays are shared with the main OS (i.e. `os_prefix` is ignored).
 
+[[sha256]]
+=== `sha256`
+
+If set to non-zero, enables the logging of SHA256 hashes for loaded files, as generated by the `sha256sum` utility. The logging output goes to the UART if enabled, and is also accessible via `sudo vcdbg log msg`. This option may be useful when debugging booting problems, but at the cost of potentially adding _many_ seconds to the boot time. Defaults to 0 on all platforms.
```

Comment:
You'd be wrong. It won't cover start.elf because that's loaded by the 2nd stage loader and the logging is generated by start.elf itself. It's for the kernel, initramfs, device tree and overlays. It does sound like it needs idiotproofing.

Reply:
We should definitely turn idiotproofing on by default in GitHub,.